pkg/virt-operator/kubevirt.go

```diff
@@ -1151,3 +1152,32 @@ func (c *KubeVirtController) syncDeletion(kv *v1.KubeVirt) error {
 	logger.Info("Processed deletion for this round")
 	return nil
 }
+
+// checkIfConfigMapStillExists This function validates that the obsolete kubevirt-config configMap is no longer exist.
+// The user should not use this configMap to configure KubeVirt, and KubeVirt will ignore it anyway. In case the configMap
+// still exists, this function emit an event to notify the user, and ask them to delete it.
+func (c *KubeVirtController) checkIfConfigMapStillExists(logger *log.FilteredLogger, trigger <-chan time.Time) {
+	ctx := context.Background()
+	getOpts := metav1.GetOptions{}
+
+	msg := fmt.Sprintf("the %s configMap is still deployed. KubeVirt does not support this configMap and it can be safely removed", obsoleteCmName)
+
+	go func() {
```

Comment:
@nunnatsa WDYT about using `sync.Once.Do` here? this function should be called only once during virt-operator init right? otherwise ignore my comment here

Reply:
Changing to another implementation as suggested by @rmohr - then it should not happen.